### ISSUE TITLE
Tensorflow seq-to-seq Arabic chatbot

### ISSUE THREAD

**Issue by EmanSaad11** [tensorflow/tensorflow#14142]
i'm working on an AI-chatbot for Arabic Language. I follow this tutorial in tensrflow seq-to-seq model. So far everything is doing great where i trained the model on my Arabic data using GPU and test the pre-trained model.But the some of them answers were correct while other are unrelated at all.

So, my questions are:

1. the model uses GRU in creating the model, should i change it to LSTM?

2. i used the same tokenizer in the (French to English translation tutorial), should i change it and use an Arabic tokenizer?

3. Even if i write an input it will reply with Arabic sentence, what i need is if the inout is new to the chatbot it should say "sorry i don't understand" for example

Basically, is there are changes should be done to make the chatbot handles the Arabic language very well

**Comment by Carmezim** [comment]
Hi @EmanSaad11, cool you're using TF to build your chatbot for arabic. These questions are better suited for StackOverflow.com though, as Github issues are reserved for bugs and feature requests. Just look/post under the TensorFlow tag. There's great community support there.

**Comment by asimshankar** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by abdallahmohammed197** [comment]
@EmanSaad11 what's the dataset that you're using? 
i am trying to build one as well but didn't land in any available corprus

**Comment by EmanSaad11** [comment]
@abdallahmohammed197 Hi , that was my biggest problems so the solution was to create my own since my Arabic Chatbot is for customer service I built a data set with questions that the users could ask anlog with thier answer . ( the questions was collected by using a simple online survey)

**Comment by abdallahmohammed197** [comment]
@EmanSaad11 is it possible to share the dataset? 
i would like to have your contact as well.  i am working in a similar problem

**Comment by abdallahmohammed197** [comment]
@EmanSaad11 is it possible to share contacts please!. i am working in something similar and could really use your corpus. i've mentiond my email above

**Comment by khaldoonH** [comment]
@EmanSaad11 hi, I am working in similar and could really use your corpus. I really need it, can you share the dataset please, my email alhussayni83@gmail.com
